recipes/skutil/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "skutil" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c054ab9359420e11507776d869ed047bbfdaf40b24f2c116ee9ef63d7b984c14
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
If you are running on >=3.5 you probably cannot build it with <3.5. Please add the same constraint it.